packages/paste-core/components/sidebar/src/navigation/styles.tsx

```diff
@@ -66,9 +64,9 @@ export const sidebarNavigationItemNestedStyles: (theme: ThemeShape) => BoxProps
   paddingY: 'space20',
   fontSize: 'fontSize20',
   flexGrow: 1,
-  marginLeft: 'spaceNegative60',
   width: `calc(100% + ${theme.space.space60})`,
```

Comment:
todo: you have to remove this now too since you removed the negative margin, otherwise the focus border is too wide:
<img width="258" alt="image" src="https://github.com/twilio-labs/paste/assets/1592327/305ede58-7e69-4443-b685-3889b95d6925">
vs
<img width="227" alt="image" src="https://github.com/twilio-labs/paste/assets/1592327/7f4d3aec-2cfa-4ae8-8667-060cde9eb0ea">